JaneSliderControl/SliderControl/SliderControl.swift

```diff
@@ -148,9 +154,24 @@ import UIKit
             case .changed:
                 guard self.shouldSlide && x > CGFloat(self.sliderWidth) && x <= self.bounds.size.width + padding else { return }
                 self.sliderWidthConstraint.constant = x
-                self.progress = Float(min(x/self.bounds.size.width, 1))
+                let progress = Float(min(x/self.bounds.size.width, 1))
+                if #available(iOS 10.0, *) {
+                    if progress > 0.65, progress > self.progress, self.hapticFeedbackIndicator != .success {
```

Comment:
Since we are using the 0.65 in multiple places now, it makes sense to move it to a constant variable instead of a magic number